src/core/worker.js

```diff
@@ -512,11 +512,16 @@ class WorkerMessageHandler {
     handler.on(
       "GetOperatorList",
       function wphSetupRenderPage(data, sink) {
-        var pageIndex = data.pageIndex;
+        const pageIndex = data.pageIndex;
         pdfManager.getPage(pageIndex).then(function (page) {
-          var task = new WorkerTask(`GetOperatorList: page ${pageIndex}`);
+          const task = new WorkerTask(`GetOperatorList: page ${pageIndex}`);
           startWorkerTask(task);
 
+          const annotationTask = new WorkerTask(
```

Comment:
I'd say this was the most critical change, as you can see I had to create another task for the parsing operator list under annotations. Obviously this brings performance concerns.

Reply:
On my list. I'm wondering if we can parse this in `Page` parsing currently.